ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -54,6 +55,12 @@ void py_module(py::module& module) {
     
     auto m_unary_backward = module.def_submodule("unary_backward", "unary_backward operations");
     unary_backward::py_module(m_unary_backward);
+    
+    auto m_ccl_all_gather = module.def_submodule("ccl_all_gather", "collective communication operations");
```

Comment:
Why? :) 

Please use single submodule ccl, no need for separate submodules for all_gather and line_all_gather.